docs/Documentation/Applications/LBMcfd.md

```diff
@@ -0,0 +1,124 @@
+---
+title: LBM-cfd
+parent: Applications
+---
+
+# Lattice Boltzmann Method based CFD applications: M-Star, marbles, Ansys Discovery
+<!---
+**Documentation:** [ link to documentation](https://nrel.gov)
+-->
+
+
+The lattice Boltzmann method (LBM) is a novel and unique simulation method whose meshless and exact conservation combines the best parts of the immersed boundary method and the finite volume method, without the hassle of mesh generation. The method completely does away with spatial discretisation in favour of advection of particles with pre-determined discrete velocities such that they hop to lattice sites on a structured grid. Consequently, the method does not incur any errors due to numerical dissipation or dispersion, thereby ensuring conservation of globally conserved properties like mass and total energy. The time discretisation is done with the trapezoidal rule, which results into transient simualations with second order accuracy in time. The LBM has been successfully applied to a range of problems in fluid dynamics including but not limited to transitional flows, flows involving complex moving geometries, compressible flows, multiphase flows, rarefied gases, combustion, electrochemical devices etc.. 
```

Comment:
I don't think we need this much explanation on the method, as that's not really the purpose of our documentation. A brief intro is okay.